### PR TITLE
feat(filters): improve help text

### PR DESCRIPTION
**Pull request by glennkitchellcaci**
Added popovers and placeholders to text-based Filters to improve understanding which which Filter types have case-sensitivity and wildcard support.

### CODE REVIEW

**Comment by bradh** [comment]
The travis failure looks related. Probably broke some assumption in the tests.

File: src/os/ui/filter/op/islikeop.js [hunk 1]
```diff
@@ -15,8 +15,14 @@ goog.require('os.xsd.DataType');
  */
 os.ui.filter.op.IsLike = function() {
   os.ui.filter.op.IsLike.base(this, 'constructor',
-      'PropertyIsLike', 'is like', 'like', [os.xsd.DataType.STRING],
-      'wildCard="*" singleChar="." escape="\\"', 'e.g. abc*');
+      'PropertyIsLike', 'is like', 'like',
+      [os.xsd.DataType.STRING], 'matchCase="false" wildCard="*" singleChar="." escape="\\"',
```
Comment (jkae) [inline src/os/ui/filter/op/islikeop.js:19]: https://github.com/ngageoint/opensphere/blob/044e0e8bea9a1e68b09b9f10b62ad4feb4314b87/test/os/ui/filter/op/islikeop.test.js#L15

Comment (glennkitchellcaci) [reply src/os/ui/filter/op/islikeop.js:19]: > The travis failure looks related. Probably broke some assumption in the tests.

Ty ty.

And... for now I'm actually going to revert this. Looks like the "matchCase" attribute isn't for PropertyIsLike (a value comparison); only the binary comparisons. https://docs.geoserver.org/latest/en/user/styling/sld/reference/filters.html

File: src/os/ui/filter/op/op.js [hunk 2]
```diff
@@ -283,3 +295,21 @@ os.ui.filter.op.Op.prototype.validate = function(value, key) {
...
+/**
+ * Text reused in several ops; to save on total the KB of the min file
+ * @enum {string}
+ * @const
+ */
+os.ui.filter.op.Op.TEXT = {
+  CASE_INSENSITIVE: '  (INFO: not case-sensitive)',
+  CASE_INSENSITIVE_DETAIL: '<strong>Case-insensitive</strong><sup>1</sup>, i.e. "A" == "a" and<br />' +
+  '<strong>Wildcard</strong> support, i.e. "a*" == "ab"<br /><br /><sup>1</sup>&nbsp;Typically, case' +
+  '-<i>in</i>sensitive text search is available for "like" filters. Some OGC Servers may not implement it.',
```
Comment (jsalankey) [inline src/os/ui/filter/op/op.js:309]: The reference to OGC servers here is not totally generic. This filter builder can be used for other server types. "Data Providers may not support it" would be a more generic description.

Comment (glennkitchellcaci) [reply src/os/ui/filter/op/op.js]: +1

**Review by jsalankey — APPROVED** [review]

**Pull request merged**